apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -110,6 +110,16 @@ ConversationAccessor.prototype = {
     this.actions.flick(conversationContainer, 50, 50, 50, 350).perform();
   },
 
+  fakeScrollUpTo: function(scrollTop) {
+    var conversationContainer = this.client.findElement(
+      SELECTORS.container
+    );
+
+    conversationContainer.scriptWith(function(container, scrollTop) {
+      container.scrollTop = scrollTop;
+    }, scrollTop);
+  },
```

Comment:
this is not a real flick, but it's a lot easier to control... tell me what you think